### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by ThijsVroegh**
Summary:
This PR updates the project documentation of Unraphael to reflect recent changes in the streamlit pages. Key updates include:

- [x] Updated the structure 
- [x] Added contents for most pages, including background, overview, and description of all the streamlit pages
- [x] These updates will improve the comprehensibility and usability of the documentation, ensuring that users have accurate and up-to-date information. It also gives further structure for ourselves

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -5,13 +5,36 @@ repo_name: GitHub
...
+  - About:
+    - How to Cite: About/CITE.md
+    - Contact information: About/CONTACT.md
+    - In action: About/ACTION.md
+    - FAQ's: About/FAQS.md
 
 theme:
-  name: material
+  name: readthedocs
+  color_mode: dark
```
Comment (stefsmeets) [inline mkdocs.yml:37]: What are your reasons for changing the theme? The material theme is one of the best things about mkdocs imo.

```suggestion
  name: material
```

**Review by stefsmeets — CHANGES_REQUESTED** [review]
Well... you have been busy writing 😅

I am a bit overwhelmed by the amount of text. I'm not sure whether it serves the purpose of documenting the code / tool. It's somewhat difficult to navigate and considering that we are still actively working on the code, this will make it difficult to keep up to date.

I was also surprised by many of the empty pages and the amount of fluff. If we have nothing to say, then we might just leave these out. Let me know what you think.

File: docs/About/CITE.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+You can do this easily with one of these options:
+
+- The 'cite this repository' menu in the right tab at https://github.com/DecodingRaphael/unraphael.
+- The [citation file](https://github.com/DecodingRaphael/unraphael/blob/main/CITATION.cff).
+- If you prefer to copypaste, here you have the APA and BibTex strings:
+ 
+<p class="mint-green-code-block">
+<b>APA</b>: Smeets, S. & Vroegh, T.(2024). Unraphael (Version 0.2.1) [Computer software].
+https://github.com/DecodingRaphael/unraphael
+</p>
```
[the remark this answers is not in the document]
Comment (ThijsVroegh) [reply docs/About/CITE.md]: That's awesome; thanks, I included it

File: docs/About/CITE.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<style>
```
[the remark this answers is not in the document]
Comment (ThijsVroegh) [reply docs/About/CITE.md]: Yes, and it looks nice!!

File: docs/About/CITE.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+<p class="mint-green-code-block">
+<b>APA</b>: Smeets, S. & Vroegh, T.(2024). Unraphael (Version 0.2.1) [Computer software].
+https://github.com/DecodingRaphael/unraphael
+</p>
+<p class="mint-green-code-block">
+<b>BibTex</b>: @software{unraphael,
+  author = {Smeets, Stef. and Vroegh, Thijs}, title = {unraphael 
+  (Version 0.2.1) [Computer software]},
+  url = {https://github.com/DecodingRaphael/unraphael}
+</p>
```
[the remark this answers is not in the document]
Comment (ThijsVroegh) [reply docs/About/CITE.md]: done

File: docs/About/FAQS.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+**FAQ's**
+
+![Unraphael banner](https://raw.githubusercontent.com/DecodingRaphael/unraphael/main/src/unraphael/data/logo.png#gh-light-mode-only)
+
+frequently asked questions about Unraphael with their answers. E.g., where can I find information about how to…?
+
+More coming soon!
```
[the remark this answers is not in the document]
Comment (ThijsVroegh) [reply docs/About/FAQS.md]: no, I deleted it

File: docs/Developers/CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+**CHANGELOG**
+
+More coming soon!
```
[the remark this answers is not in the document]
Comment (ThijsVroegh) [reply docs/Developers/CHANGELOG.md]: I deleted it!

File: docs/Processing/ANALYSIS.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+(with A in amount of pixels)
+
+This then allows us to answer how similar the areas of the main outlines in two paintings are, after accounting for differences in the sizes of the paintings and their corresponding photos.
+
+
+### Steps to Follow
+
+1. **Upload Images and Real Sizes:**
+   - **Upload Photos:** Start by uploading the digital photos of the paintings.
+   - **Upload Real Sizes:** Upload an Excel file containing the real dimensions of the paintings in centimeters.
```
[the remark this answers is not in the document]
Comment (ThijsVroegh) [reply docs/Processing/ANALYSIS.md:29]: @stefsmeets . I worked on this last week, see the ratio branch. It works nice when no alignment is chosen. I still have to talk to Elena what the consequence of alignment is for this anlysis

File: docs/Started/ISSUES.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+2
+
+……
+
+3
+
+……
+
+This list of issues can be further updated in the future.
```
[the remark this answers is not in the document]
Comment (ThijsVroegh) [reply docs/Started/ISSUES.md]: yese, I agree

File: docs/Started/TUTORIALS.md [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+3. **Analyze Differences**: Conversely, areas of low similarity might highlight unique features or variations between the images, which could be of interest depending on the focus of your analysis.
+
+---
+
+## Conclusion
+
+By following these steps—preprocessing, segmenting, and aligning images—you can effectively prepare your images for detailed analysis. The heatmap generated during the alignment step is particularly valuable for uncovering structural similarities between images, offering insights that can inform your research or analysis. Whether you're working with art history images or other types of visual data, this process will help you standardize and compare your images systematically.
+
+
+## Clustering Images with the Streamlit Application
```
[the remark this answers is not in the document]
Comment (ThijsVroegh) [reply docs/Started/TUTORIALS.md]: done

File: mkdocs.yml (hunk 1 above)
Comment (ThijsVroegh) [reply mkdocs.yml]: I changed it back again. The structure is, indeed, nice! Perhaps choose later a different color then purple?

File: docs/Started/INSTALLATION.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
+# Installation
+
+To install:
+
```
[the remark this answers is not in the document]
Comment (ThijsVroegh) [reply docs/Started/INSTALLATION.md]: I made a reference to the readme to avoid updating it twice

File: mkdocs.yml (hunk 1 above)
Comment (stefsmeets) [reply mkdocs.yml]: I have no strong opinion on the color 😅

File: docs/Processing/ANALYSIS.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (stefsmeets) [reply docs/Processing/ANALYSIS.md:29]: Okay, but please choose something different than Excel as data format.

File: mkdocs.yml [hunk 11]
```diff
@@ -5,13 +5,33 @@ repo_name: GitHub
...
+    - Code of Conduct: Developers/CODE_OF_CONDUCT.md
+    - Contributing guidelines: Developers/CONTRIBUTING.md
+  - About:
+    - How to cite: About/CITE.md
+    - Contact information: About/CONTACT.md
+    - In action: About/ACTION.md
 
 theme:
   name: material
+  #color_mode: dark
```
Comment (stefsmeets) [inline mkdocs.yml:34]:
```suggestion
```

File: docs/Started/INSTALLATION.md [hunk 12]
```diff
@@ -0,0 +1,3 @@
+# Installation
+
+See the README.md file in the root of the repository for installation instructions.
```
Comment (stefsmeets) [inline docs/Started/INSTALLATION.md:3]: This page feels a bit pointless

File: docs/About/ACTION.md [hunk 13]
```diff
@@ -0,0 +1,7 @@
+![Unraphael banner](https://raw.githubusercontent.com/DecodingRaphael/unraphael/main/src/unraphael/data/logo.png#gh-light-mode-only)
+
+This section includes links to projects, events, papers, blogposts, videos, etc. which mention or make use of Unraphael.
+
+**Blogs**
+
+More coming soon!
```
Comment (stefsmeets) [inline docs/About/ACTION.md:7]: I suggest removing this page as well.

**Comment by stefsmeets** [comment]
I don't know @ThijsVroegh , I don't think this structure is all that great. It's a lot of text and I'm not sure what it is documenting. It does not link to any of the pages that we have in the application, and if I want help on any of the pages, where do I go?

Crucially, it is missing installation and usage instructions. We have these front and center in the current version of the docs: https://unraphael.readthedocs.io/en/latest/

I think this is a step backwards.

**Comment by stefsmeets** [comment]
I revised the documentation addressing some of my own comments. I also flattened the layout to make it a bit easier to navigate, and removed some of the superfluous pages.

**Pull request merged**